web/src/features/panels/ranking-panel/ZoneList.tsx

```diff
@@ -0,0 +1,44 @@
+import type { ReactElement } from 'react';
+import { GridState } from 'types';
+import { CountryTag } from '../zone/CountryTag';
+
+interface ZonelistProperties {
+  data: ZoneRow[];
+}
+
+export interface ZoneRow {
+  zoneId: keyof GridState;
+  ranking?: number | undefined;
```

Comment:
```suggestion
  ranking?: number;
```
We don't need the undefined type if it's optional.